doc/ref_model/chapters/chapter08.md

```diff
@@ -94,6 +94,59 @@ Figure 8-3 shows a conceptual Telco Operator Platform Architecture. The Cloud In
+
+As outlined in "Figure 8-3 - Conceptual Architecture of a Telco Operator Platform" above the exposure point for the Management Interactions is the "Cloud Service Broker" and the "Cloud Service Broker". The set of interactions that these interface points need to provide are defined by the "Multii-Cloud Interaction Model". This provides a taxonomy for the interactions and is illustrated below:
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-02.png" alt="Multi-Cloud Interactions Model"></p>
+<p align="center"><b>Figure 8-4:</b> Multi-Cloud Interactions Model</p>
+
+The model defines the following core roles:
+- Communications Service Provider (CSP) - is the party responsible for providing end user service to their customer
+- Customer / User - are the parties that use the service (User) and establishes the business agreement for the service provision (Customer). For retail services the customer and user are the same party, while for enterprise services the Enterprise is the Customer (responsible for the business agreement) and its representatives are the Users.
+- Cloud Providers - are the parties providing the cloud services. These services could be any XaaS service. It could be that a CSP has an agreement with a SaaS Cloud, which in turn uses an IaaS Cloud Provider to deliver their service.
+
+The set of high level interactions cover:
+- Manage Account - covering Account, Users, Billing
+- Manage Connectivity - Public or Private Network, VPN Configuration, CSP Edge / Cloud Connection Configuration, Connection Security Profile
+- Manage Resource - Resource Pool Management, VM / VNF Management (CPU, Memory, Storage, Network), Image Repository Management, Storage Management, VNF / CNF LCM, Monitor Resources
+- Manage App/VNF - Image / Container / Registry Management, Deploy/Configure/Scale/Start/Stop App/VNF, Monitor App/VNFs
+- Transactions / Conversations - Use Communications Services, Use Edge Applications Services, Use Cloud Services
+
+<a name="8.2.4.2"></a>
+### 8.2.4.2 Stereo-Typical Scenarios
+A set of stereo-typical interactions cases are illustrated for the cases of a Simple Infrastructure-as-a-Service (IaaS) and Software-as-a-Service (SaaS), where deployment is on a Cloud Provider's centralised sites and/or Edge sites. The scenarios help highlight needs for the Cloud Service Broker and Cloud Resources Broker (as per Figure 8.3) and hence extent of orchestration required to manage the interactions.
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-simple-stereo-types-02.png" alt="Simple Stereo-Type Interactions"></p>
+<p align="center"><b>Figure 8-5:</b> Simple Stereo-Type Interactions</p>
+
+The following patterns are visible:
+- For IaaS Cloud Integration:
+  - Cloud behaves like VIM and so needs Orchestration
```

Comment:
typo
```suggestion
  - Cloud behaves like a set of virtual servers and, thus, requires virtual server life-cycle management and orchestration
```

Reply:
Thank you @pgoyal01 , agree good to avoid acronyms, especially when undefined.